components/appointments/id/index.tsx

```diff
@@ -112,160 +163,328 @@ export default function Appointment({ aid, session }: AppointmentProps) {
+              </h2>
             </CardHeader>
-            <CardBody>
-              <div className="flex items-center">
+            <Divider className="border-dashed border-divider" />
+            <CardBody className="space-y-4 p-6">
+              <div className="flex max-w-xl items-center justify-between">
                 <div>
-                  <Image
-                    src={'/assets/placeholder-avatar.jpeg'}
-                    alt={appointment.patient.name}
-                    width={100}
-                    height={100}
-                    className="rounded-full text-slate-300"
+                  <CellRenderer
+                    label="Date & Time"
+                    icon="solar:calendar-bold-duotone"
+                    value={format(
+                      new Date(appointment.date),
+                      'MMM d, yyyy - h:mm a'
+                    )}
+                    classNames={{ icon: 'text-yellow-500 bg-yellow-50' }}
                   />
-                </div>
-                <div className="">
                   <CellRenderer
-                    label="Name"
-                    icon="cbi:abc"
-                    value={appointment.doctor?.name}
-                    classNames={{ icon: 'text-pink-500 bg-pink-50' }}
-                    className="text-lg text-black"
+                    label="Appointment Type"
+                    icon="solar:stethoscope-bold"
+                    value={appointment.type}
+                    classNames={{
+                      icon: 'text-primary bg-primary-50',
+                    }}
+                  />
+                  <CellRenderer
+                    label="created by"
+                    icon="solar:people-nearby-bold"
+                    value={appointment.createdBy || 'System'}
+                    classNames={{
+                      icon: 'text-purple-500 bg-purple-50',
+                    }}
                   />
+                </div>
+                <div>
                   <CellRenderer
-                    label="Email"
-                    icon="material-symbols:mail"
-                    value={appointment.doctor?.email}
+                    label="Mode"
+                    icon="solar:hospital-bold"
+                    value={appointment.additionalInfo.type}
                     classNames={{ icon: 'text-blue-500 bg-blue-50' }}
-                    className="text-lg text-black"
                   />
                   <CellRenderer
-                    label="Phone Number"
-                    icon="ic:baseline-phone"
-                    value={appointment.doctor?.phone}
-                    classNames={{ icon: 'text-green-500 bg-green-50' }}
-                    className="text-lg text-black"
+                    label="Created At"
+                    icon="solar:clock-circle-linear"
+                    value={
+                      appointment.createdAt
+                        ? formatDate(new Date(appointment.createdAt), 'PPP')
```

Comment:
use `format` only no need of `formatDate`
```suggestion
                        ? format(new Date(appointment.createdAt), 'PPP')
```